BlueprintUI/Tests/XCTestCase+AppHost.swift

```diff
@@ -0,0 +1,94 @@
+import BlueprintUI
+import UIKit
+import XCTest
+
+
+extension XCTestCase {
+
+    ///
+    /// Call this method to show a view controller in the test host application
+    /// during a unit test. The view controller will be the size of host application's device.
+    ///
+    /// After the test runs, the view controller will be removed from the view hierarchy.
+    ///
+    /// A test failure will occur if the host application does not exist, or does not have a root view controller.
+    ///
+    public func show<ViewController: UIViewController>(
+        vc viewController: ViewController,
+        loadAndPlaceView: Bool = true,
+        test: (ViewController) throws -> Void
+    ) rethrows {
+
+        var temporaryWindow: UIWindow? = nil
+
+        func rootViewController() -> UIViewController {
```

Comment:
The two cases in here ensure we can run the tests in both cocoapods world, which provides an app host, and SPM, which does not (there is no option in SPM to provide an app host).

Reply:
I'm trying to make it work/exist as close to as possible to a "real" UI – hence pulling in the app host changes. While it passed before, there's enough special UIKit behavior that exists when being in a real app that I'd rather we just mirror real apps as much as possible.